byterun/caml/roots.h

```diff
@@ -35,10 +35,48 @@ CAMLextern void caml_do_local_roots (scanning_action, value *, value *,
 CAMLextern void caml_do_local_roots(scanning_action f, char * bottom_of_stack,
                                     uintnat last_retaddr, value * gc_regs,
                                     struct caml__roots_block * local_roots);
-#endif
+#endif /* NATIVE_CODE */
```

Comment:
If you want to put this annotation on `#endif` you should avoid confusion: invert the condition and exchange the `then` and `else` parts.